src/Components/FlashSalesCountdownSection/Countdown/Countdown.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './Countdown.module.css';
+const Countdown = ({ days, hours, minutes, seconds }) => {
+  const countdownUnits = [
+    { label: 'Days', value: days, counterClass: styles.DaysCounter, labelClass: styles.days, valueClass: styles.heading },
+    { label: 'Hours', value: hours, counterClass: styles.HoursCounter, labelClass: styles.hours, valueClass: styles.heading },
+    { label: 'Minutes', value: minutes, counterClass: styles.MinutesCounter, labelClass: styles.minutes, valueClass: styles.heading },
+    { label: 'Seconds', value: seconds, counterClass: styles.SecondsCounter, labelClass: styles.seconds, valueClass: styles.heading }
+  ];
+  const semiCircleClasses = [
+    { cloneClass: styles.semiClone1, ellipse1: styles.ellipse18, ellipse2: styles.ellipse19 },
+    { cloneClass: styles.semiClone2, ellipse1: styles.ellipse19, ellipse2: styles.ellipse20 },
+    { cloneClass: styles.semiClone3, ellipse1: styles.ellipse21, ellipse2: styles.ellipse22 }
+  ];
```

Comment:
I would suggest to format the data (as what you expect from a beautified JSON file), and move it to the correct directory src/data.

Reply:
done.